.gitignore

```diff
@@ -14,3 +14,6 @@ locale/**/*.mo
 .tox
 deis
 locale/templates/LC_MESSAGES/django.po
+stunnel.log
+stunnel/*
```

Comment:
Can we change this to `stunnel/stunnel.pem`? Being explicit can help save us from "ARGH! Why is this magically NOT working with this 1 filename?!" moments.